cinder/volume/drivers/nexenta/nfs.py

```diff
@@ -572,16 +573,16 @@ def _create_regular_file(self, nms, path, size):
         :param path: path to new file
         :param size: size of file
         """
-        block_size_mb = 1
-        block_count = size * units.Gi / (block_size_mb * units.Mi)
+        BLOCK_SIZE_MB = 1
```

Comment:
You have already assigned 1 to the 'constant' in the beginning of file :) I think this line can be removed.